.travis.yml

```diff
@@ -2,6 +2,5 @@ language: node_js
 os: linux
 dist: xenial
 node_js:
-  - "16"
```

Comment:
Why node 16?

Reply:
Removing it because it looks like mocha tests are failing due to incompatible versions. Will have to make a major upgrade of mocha here and dig more deeply into it as we prepare for Meteor on Node 16, but that is not necessary at this point in time as the main focus here is to update Typescript for inclusion in Meteor 2.5.